src/Basis/Constructors.jl

```diff
@@ -2,28 +2,151 @@
 # finite element bases
 # ------------------------------------------------------------------------------
 
+using Polynomials
```

Comment:
I think this is no longer used.

Reply:
from checking today, I think we still need it in here.